tools/cli/helpers/projectHelpers.js

```diff
@@ -19,9 +19,23 @@ export const projectTypes = [ 'github-actions', 'packages', 'plugins' ];
 export function allProjects() {
 	let projects = [];
 	projectTypes.forEach( type => {
```

Comment:
Instead of hard-coding a list of `projectTypes`, you could just look for anything matching `./projects/*/*/composer.json`. Strip the leading `./projects/` and trailing `/composer.json` and you have your slug, and types are easy to derive from there too. I've started using that approach in a bunch of the other tooling I've been writing.

But maybe that's an improvement for a future PR.

Reply:
I noticed that in one of your PRs. Since we decided that all projects need a composer.json, that would work. The very first iteration I had was to look for folders, but since we had `editor-extensions` created as a "future use" case, it broke it.

The main reason I need types at this point is the project picker (e.g. `jetpack install` without any args) looks at the types array, and asks that to save some scrolling. I could probably switch to something that suppports completion or something to make that better.